### ISSUE TITLE
Questions about tau-norm with randaugment

### ISSUE THREAD

**Issue by adf1178** [dvlab-research/Parametric-Contrastive-Learning#3]
Thanks for your exciting work!
Tau-norm with randaugment performs so well as shown in Table 3 and Table 5. I wonder about its implementation, just use **augmentation_randncls** as train_transform in training stage-1?

**Comment by jiequancui** [comment]
Hi,
Thank you for your questions!
For Tau-norm method, in training stage-1. we use standard augmentations following the original paper (https://github.com/facebookresearch/classifier-balancing) besides the randaugment. Just like that:
          [ transforms.RandomResizedCrop(224),
            transforms.RandomHorizontalFlip(),
            transforms.ColorJitter(brightness=0.4, contrast=0.4, saturation=0.4, hue=0),
            randaugment,
            transforms.ToTensor(),
            normalize, ]

**Comment by adf1178** [comment]
Thanks for your reply!
Now this i my implementation.
`   
 augmentation_randncls = [

            transforms.RandomResizedCrop(224, scale=(0.08, 1.)),
            transforms.RandomHorizontalFlip(),
            transforms.RandomApply([
                transforms.ColorJitter(0.4, 0.4, 0.4, 0.0)
            ], p=1.0),
            rand_augment_transform('rand-n{}-m{}-mstd0.5'.format(2, 10), ra_params),
            transforms.ToTensor(),
            normalize,
    ]
    train_transform = transforms.Compose(augmentation_randncls)
`
Is that right?

**Comment by jiequancui** [comment]
Yes, it is.

**Comment by blue-blue272** [comment]
> Hi,
> Thank you for your questions!
> For Tau-norm method, in training stage-1. we use standard augmentations following the original paper (https://github.com/facebookresearch/classifier-balancing) besides the randaugment. Just like that:
> [ transforms.RandomResizedCrop(224),
> transforms.RandomHorizontalFlip(),
> transforms.ColorJitter(brightness=0.4, contrast=0.4, saturation=0.4, hue=0),
> randaugment,
> transforms.ToTensor(),
> normalize, ]

Hi, I have another question. What are the implementation details for tau-norm, for example, the learning rate, batch size, and training epoch?

**Comment by jiequancui** [comment]
Hi,
We follow the tradition. 

learning rate 0.05, batch size 128, training epoch 400.
